homework/style.css

```diff
@@ -1,3 +1,179 @@
+* {
+  box-sizing: border-box;
+  font-family: Roboto;
+}
+
+#root {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
 .alert-error {
-  color: red;
+  color: #ffffff;
+  background-color: #b00020;
+  width: 100%;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  padding-left: 25px;
+  margin-top: 10px;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.main-header {
+  width: 70%;
+  background-color: #5e35b1;
+  color: white;
+  font-weight: bold;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  padding-left: 25px;
+}
+
+#main {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 70%;
+}
+
+section {
+  width: 49%;
+  margin-top: 5px;
+}
+
+.header-text {
+  float: left;
+}
+
+.repo-select {
+  margin-left: 20px;
+  width: 200px;
+  border-radius: 3px;
+}
+
+.repo-select:focus {
+  border-color: #ffffff;
+}
+
+.contributions-section {
+  box-shadow: 0px 0px 4px #7e57c2;
+  border-radius: 3px;
+}
+
+table {
+  box-shadow: 0px 0px 4px #7e57c2;
+  padding-left: 25px;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  width: 100%;
+  margin-bottom: 5px;
+  line-height: 1.5em;
+  border-radius: 3px;
+}
+
+table:hover {
+  box-shadow: 0px 0px 8px #311b92;
+}
+
+.td-header {
+  font-weight: bold;
+  margin-left: 30px;
+  width: 12%;
+}
+
+ul {
+  list-style: none;
+  padding-left: 0px;
+  margin-top: 0px;
+  margin-bottom: 0px;
+}
+
+ul p {
+  padding-left: 10px;
+  color: #344955;
+}
+.list-item {
+  display: flex;
+  width: 100%;
+  border-bottom: 1px solid #eeeeee;
+  align-items: center;
+}
+
+.list-item:hover {
+  box-shadow: 0px 0px 8px #311b92;
+}
+
+.contributor-img {
+  width: 15%;
+  border-radius: 4px;
+  margin: 10px 30px 10px 10px;
+}
+
+.list-item a {
+  text-decoration: none;
+  width: 75%;
+}
+
+.contributions-div {
+  background-color: #03dac5;
+  border-radius: 4px;
+  color: #ffffff;
+  width: 10%;
+  text-align: center;
+  margin-right: 10px;
+}
+
+@media screen and (min-width: 600px) and (max-width: 1200px) {
+  .main-header {
+    width: 95%;
+  }
+
+  #main {
+    width: 95%;
+  }
+
+  ul {
+    width: 95%;
```

Comment:
If I remove this line the hover outline now takes the full width of the contributions section (see picture above), which looks a bit better.

Reply:
I should have deleted it, it belongs to ul from week 1. I did so now and will push the change with week3 homework.